### PR TITLE
Final swri integration

### PR DESCRIPTION
**Pull request by marrts**
Changes made including #75 and getting the system working with robot communications at SwRI

### CODE REVIEW

File: crs_motion_planning/src/motion_planning_server.cpp [hunk 1]
```diff
@@ -264,28 +180,33 @@ class MotionPlanningServer : public rclcpp::Node
   void planProcess(std::shared_ptr<crs_msgs::srv::PlanProcessMotions::Request> request,
                    std::shared_ptr<crs_msgs::srv::PlanProcessMotions::Response> response)
   {
-    crs_motion_planning::pathPlanningConfig motion_planner_config = *motion_planner_config_;
+    namespace fs = boost::filesystem;
+    crs_motion_planning::pathPlanningConfig::Ptr motion_planner_config;
+    std::string config_rel_path = "config/motion_planning/MP_config.yaml";
```
Comment (jrgnicho) [inline crs_motion_planning/src/motion_planning_server.cpp]: This path should be passed as a parameter?

File: crs_process_data/data/main/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  process_execution:
+    time_tolerance: 5.0
+    joint_tolerance: [0.035, 0.035, 0.035, 0.035, 0.035, 0.035]
+  part_registration:
+    simulation_pose: [-0.7, 0.0, 1.1, 0.0, 0.0, 1.57]
+    seed_pose: [-0.68, 0.0, 1.0, 0.0, 0.0, 1.57]
+    target_frame_id: world
+    part_file: /home/jnicho/crs_data/part_2/part_2.stl
+    toolpath_file: /home/jnicho/crs_data/part_2/job_0_degrees_20200520T192241.997282.yaml
+  part_rework: {}
```
Comment (jrgnicho) [inline crs_process_data/data/main/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml]: Not sure how this file made it here, unless you think it is necessary it should be removed.

File: crs_process_data/data/surrogate1/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  process_execution:
+    time_tolerance: 5.0
+    joint_tolerance: [0.035, 0.035, 0.035, 0.035, 0.035, 0.035]
+  part_registration:
+    simulation_pose: [-0.7, 0.0, 1.1, 0.0, 0.0, 1.57]
+    seed_pose: [-0.68, 0.0, 1.0, 0.0, 0.0, 1.57]
+    target_frame_id: world
+    part_file: /home/jnicho/crs_data/part_2/part_2.stl
+    toolpath_file: /home/jnicho/crs_data/part_2/job_0_degrees_20200520T192241.997282.yaml
+  part_rework: {}
```
Comment (jrgnicho) [inline crs_process_data/data/surrogate1/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml]: Removed file if not used

**Review by jrgnicho — COMMENTED** [review]
Looks fine for the most part, I'll make the changes and push to your branch and then merge into integration, should #75 be closed?

**Comment by marrts** [comment]
Yeah, you can go ahead and close #75. I left it open since there were still a couple unresolved comments

File: crs_motion_planning/src/motion_planning_server.cpp (hunk 1 above)
Comment (marrts) [reply crs_motion_planning/src/motion_planning_server.cpp]: Changed to use a static const string defined at the top of the file, eventually I think it should be a default value that can be overwritten by the service call that uses the motion planner

File: crs_process_data/data/main/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml (hunk 2 above)
Comment (marrts) [reply crs_process_data/data/main/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml]: fixed

File: crs_process_data/data/surrogate1/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml (hunk 3 above)
Comment (marrts) [reply crs_process_data/data/surrogate1/toolpaths/part_2/configs/job_0_degrees_20200520T192241.997282_config.yaml]: fixed

**Comment by jrgnicho** [comment]
@marrts I've created PR #82 with your changes as well as some minor tweaks I made to address the comments here. So I'll close this one and #75

**Pull request closed without merging**